packages/channel-client/src/channel-client.ts

```diff
@@ -159,10 +272,25 @@ export class ChannelClient implements ChannelClientInterface {
     });
   }
 
+  /**
+   * Requests the latest budget for this site
+   *
+   * @param hubAddress The address of a state channel hub
+   * @returns A promise that resolves to a ChannelResult.
+   *
+   */
   async getBudget(hubAddress: string): Promise<DomainBudget> {
     return this.provider.send({method: 'GetBudget', params: {hubAddress}});
   }
 
+  /**
+   * Requests the funds to be withdrawn from this site's ledger channel
+   *
+   * @param hubAddress The address of a state channel hub
+   * @param hubOutcomeAddress An ethereum account that the hub's funds will be paid out to TODO this doesn't make sense
+   * @returns A promise that resolves to a DomainBudget.
```

Comment:
❓ 
(for `closeAndWithdraw`, I don't understand why I should get to say anything about the hub's outcome address)

Reply:
This probably also does not belong, but we should double check to see what (if anything) the wallet does with it.

Reply:
TODO

Reply:
#2306